Dockerfile

```diff
@@ -43,5 +46,9 @@ COPY . .
 RUN ./node_modules/.bin/gulp build --production && \
     ./manage.py collectstatic --noinput
 
+# Project permissions
+RUN  chown -R app: /home/app
```

Comment:
_not a blocker_

Does the application write to any of the folders?

If not, we don't need to do this. The downside of doing a wide-sweep `chown` is that it essentially adds the size of the `/home/app` again as a new layer, increasing the overall image size.